scripts/build_test.py

```diff
@@ -1262,7 +1262,15 @@ def test_build_js_files_in_dev_mode_with_exception_raised(self) -> None:
             build, 'main', lambda *_, **__: None,
             expected_kwargs=[{'args': []}]))
         self.exit_stack.enter_context(self.swap_with_checks(
-            sys, 'exit', lambda _: None, expected_args=[(return_code,)]))
+            sys, 'exit', lambda _: None,
+            expected_args=[
```

Comment:
Why did this change? Maybe a comment explaining the expected value might be useful.

Reply:
I removed an unreachable `return` statement from `run_webpack_compilation` in `scripts/build.py` ([diff](https://github.com/oppia/oppia/pull/18165/files#diff-c786cc346fc8fdc17f1c85f11bc187c153a3511ae549390185c7c39343d2c95dL214)). The return statement was unreachable because of a `sys.exit` call, but in the test, `sys.exit` is mocked, so now it gets called multiple times. Added a comment explaining this